_schemaV2.graphql

```diff
@@ -2163,20 +2163,16 @@ type ArtistRelatedData {
   ): ArtistConnection
 }
 
-type ArtistSeries {
-  # List of Mongo IDs for associated Artists
+type ArtistSeries implements Node {
   artistIDs: [String!]!
-  artists(page: Int, size: Int): [Artist]
-  artworkIDs: [ID!]!
-  artworksConnection(after: String, first: Int): ArtworkConnection
+  artists(page: Int, size: Int): [Artist!]!
```

Comment:
non-null list of non-null artists -> this got stricter!

Reply:
this got way better for the clients!